node/src/key_events.rs

```diff
@@ -245,7 +314,7 @@ pub struct ResharingArgs {
 
 /// Handles the follower side of a key resharing task.
 /// - Waits for a new network task channel from `channel_receiver`.
-/// - Ignores non-resharing tasks until a resharing task is received.
+/// - Drops the channel and bails if task id other than resharing is received.
```

Comment:
@robin-near : Can you check if this is correct?

In a pytest, I observed this in one of the follower nodes:
```
[2m2025-03-26T11:17:19.900879Z[0m [32m INFO[0m [2mmpc_node::key_events[0m[2m:[0m Expected Key resharing task id, received: EcdsaTaskId(ManyTriples { start: UniqueId(0, 1742987670, 0), count: 64 }); ignoring.
```
There were 80 millions lines like this in stdout before the pytest timed out.